templates/panda_harvester.cfg.rpmnew.template

```diff
@@ -123,6 +123,22 @@ autoBlacklist = False
 
 
 
+##########################
```

Comment:
Can you change them as plugin parameters? e.g. HTCondor plugin at aipanda170 has

                "submitter": {
                        "name":"HTCondorSubmitter",
                        "module":"pandaharvester.harvestersubmitter.htcondor_submitter",
                        "useAtlasGridCE": true,
                        "CEtemplateDir":"/data/atlpan/harvester_common/htcondor_atlas-grid-ce_push.sdf.d",
                        "x509UserProxy": "/data/atlpan/proxy/x509up_u25606_prod",
                        "logDir": "/data/atlpan/condor_logs",
                        "logBaseURL": "https://aipanda170.cern.ch/condor_logs",
                        "nProcesses": 4
                },

Log dir and URL can be queue/plugin specific and local httpd is not always available.